### DISCUSSION TITLE
Has the stock API changed in the master in the last month?

### DISCUSSION THREAD

**Discussion by StormTrooper** [inventree/InvenTree#6401] [Q&A] [answered]
I've got a custom php script running which queries the stock that's changed in the last 24 hours using a API.

Was running the dev version from around mid December 2023. Have now updated to the latest master and my script is failing saying the stock API endpoint is not longer valid. 

Has something changed in the way this is called?

Getting the same thing on the demo site so I don't think it's my upgrade that's borked. 
 
I was calling:
http://inventree.home:8000/api/stock/*?format=json&updated_after=2024-02-04

And get: API endpoint not found

What's this been replaced with?

**Comment by matmair** [comment, marked as answer]
I am amazed that it worked with the star, try without

**Reply by StormTrooper** [reply]
Thanks, that did the trick.  Working again.

**Reply by StormTrooper** [reply]
Had to remove the star from this as well:

http://inventree.home:8000/api/user/token/*?format=json

**Reply by wolflu05** [reply]
I guess it's related to #6251 because there everything that's not a number (the pk) got matched with `^.*$` the index route.
